cmd/minishift/cmd/start_preflight.go

```diff
@@ -231,23 +243,34 @@ func checkKvmDriver() bool {
 	return true
 }
 
-// checkHypervDriver returns true if Virtual Switch has been selected
-func checkHypervDriver() bool {
+// checkHypervDriverSwitch returns true if Virtual Switch has been selected
+func checkHypervDriverSwitch() bool {
 	switchEnv := os.Getenv("HYPERV_VIRTUAL_SWITCH")
 	if switchEnv == "" {
 		return false
 	}
 
+	return true
+}
+
+//checkHypervDriverInstalled returns ture if Hyper V driver is installed
```

Comment:
Space after `//` and hyphen in `Hyper-V`

Reply:
`ture` => `true`